tests/spec/fs.writeFile-readFile.spec.js

```diff
@@ -99,3 +99,118 @@ describe('fs.writeFile, fs.readFile', function() {
     });
   });
 });
+
+
+
+
+
+describe('fs.promises.writeFile, fs.promises.readFile', function() {
+  beforeEach(util.setup);
+  afterEach(util.cleanup);
+
+  it('should be a function', function() {
+    var fs = util.fs();
+    expect(fs.promises.writeFile).to.be.a('function');
+    expect(fs.promises.readFile).to.be.a('function');
+  });
+
+  it('should return a promise', function() {
+    var fs = util.fs();
+    var contents = 'This is a file.';
+
+    expect(fs.promises.writeFile('/myfile', contents)).to.be.a('Promise');
+    expect(fs.promises.readFile('/myfile', 'utf8')).to.be.a('Promise');
+  });
+
+  it('should error when path is wrong to readFile', function(done) {
+    var fs = util.fs();
+
+    
+    fs.promises.readFile('/no-such-file', 'utf8')
+      .then(data => expect(data).not.to.exist)
+      .catch(error => { expect(error).to.exist; expect(error.code).to.equal('ENOENT'); done(); });
+    
+
+  });
+
+  it('should write, read a utf8 file without specifying utf8 in writeFile', function(done) {
+    var fs = util.fs();
+    var contents = 'This is a file.';
+
+    fs.promises.writeFile('/myfile', contents)
+      .then( () => {
+
+        fs.promises.readFile('/myfile', 'utf8')
+          .then( (data, error) => { expect(data).to.equal(contents); expect(error).not.to.exist; done(); });
```

Comment:
Break this up into multiple lines.  Also, the `error` object won't ever get passed here, it will end up in the `catch`